Lib/ntpath.py

```diff
@@ -605,10 +593,8 @@ def _abspath_fallback(path):
 # Return an absolute path.
 try:
     from nt import _getfullpathname
-
```

Comment:
Please revert this change. Any stylistic changes would need to go in a separate PR.